src/v3/pages/reports/meeting_attendance/components/week_box/index.types.tsx

```diff
@@ -0,0 +1,9 @@
+import { TFunction } from 'i18next';
+
+export type WeekBoxProps = {
+  weekNumber: number;
+  weekend: boolean;
+  onChange: (weekNumber: number, weekend: boolean, value: number) => void;
+  t: TFunction<'translation', undefined>;
```

Comment:
@Logan08691: we should not need this. Directly use the `t` function from the hook. Thanks.